### PR TITLE
upgrade prettier

### PR DESCRIPTION
**Pull request by stas-vilchik**

### CODE REVIEW

File: server/sonar-web/src/main/js/api/issues.js [hunk 1]
```diff
@@ -72,7 +72,7 @@ export function getTags(query: {}): Promise<*> {
 
 export function extractAssignees(facet: Array<{ val: string }>, response: IssuesResponse) {
   return facet.map(item => {
-    const user = response.users ? response.users.find(user => user.login = item.val) : null;
+    const user = response.users ? response.users.find(user => user.login === item.val) : null;
```
Comment (stas-vilchik) [inline server/sonar-web/src/main/js/api/issues.js]: nice catch here

File: server/sonar-web/src/main/js/app/components/Landing.js [hunk 2]
```diff
@@ -25,10 +25,8 @@ import { getCurrentUser } from '../../store/rootReducer';
 
 class Landing extends React.PureComponent {
   static propTypes = {
-    currentUser: React.PropTypes.oneOfType([
-      React.PropTypes.bool,
-      React.PropTypes.object
-    ]).isRequired
+    currentUser: React.PropTypes.oneOfType([React.PropTypes.bool, React.PropTypes.object])
+      .isRequired
```
Comment (stas-vilchik) [inline server/sonar-web/src/main/js/app/components/Landing.js]: so so

**Comment by stas-vilchik** [comment]
@gregaubert one more time 😉

File: server/sonar-web/src/main/js/apps/settings/utils.js [hunk 3]
```diff
@@ -84,7 +84,7 @@ export function getEmptyValue(definition) {
 
   if (definition.type === TYPE_PROPERTY_SET) {
     const value = {};
-    definition.fields.forEach(field => value[field.key] = getEmptyValue(field));
+    definition.fields.forEach(field => (value[field.key] = getEmptyValue(field)));
```
Comment (SonarTech) [inline server/sonar-web/src/main/js/apps/settings/utils.js]: MAJOR Extract the assignment of "value[field.key]" from this expression. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AAssignmentWithinCondition)

File: server/sonar-web/src/main/js/apps/coding-rules/facets/repository-facet.js [hunk 4]
```diff
@@ -44,7 +44,7 @@ export default CustomValuesFacet.extend({
 
   getLabelsSource() {
     const source = {};
-    this.options.app.repositories.forEach(repo => source[repo.key] = repo.name);
+    this.options.app.repositories.forEach(repo => (source[repo.key] = repo.name));
```
Comment (SonarTech) [inline server/sonar-web/src/main/js/apps/coding-rules/facets/repository-facet.js]: MAJOR Extract the assignment of "source[repo.key]" from this expression. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AAssignmentWithinCondition)

File: server/sonar-web/src/main/js/apps/projects/store/actions.js [hunk 5]
```diff
@@ -68,37 +68,34 @@ const FACETS = [
...
 
-    const toStore = {};
+  const toStore = {};
 
-    // fill store with empty objects for expected projects
-    // this is required to not have "null"s for provisioned projects
-    expectedProjectKeys.forEach(projectKey => toStore[projectKey] = {});
+  // fill store with empty objects for expected projects
+  // this is required to not have "null"s for provisioned projects
+  expectedProjectKeys.forEach(projectKey => (toStore[projectKey] = {}));
```
Comment (SonarTech) [inline server/sonar-web/src/main/js/apps/projects/store/actions.js]: MAJOR Extract the assignment of "toStore[projectKey]" from this expression. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AAssignmentWithinCondition)

**Comment by SonarTech** [comment]
SonarQube analysis reported 3 issues
* ![MAJOR][MAJOR] 3 major

Watch the comments in this conversation to review them.

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

**Pull request merged**